capplets/windows/mate-window-properties.c

```diff
@@ -82,19 +87,24 @@ typedef struct
 } MouseClickModifier;
 
 static GtkWidget *dialog_win;
-static GtkWidget *compositing_checkbutton;
-static GtkWidget *compositing_fast_alt_tab_checkbutton;
-static GtkWidget *allow_tiling_checkbutton;
+
+/* General */
 static GtkWidget *center_new_windows_checkbutton;
```

Comment:
I wonder wether there are some GtkWidget* variables that can be local variables, like `center_new_windows_checkbutton`:
```diff
diff --git a/capplets/windows/mate-window-properties.c b/capplets/windows/mate-window-properties.c
index b2053e6..d66480c 100644
--- a/capplets/windows/mate-window-properties.c
+++ b/capplets/windows/mate-window-properties.c
@@ -89,7 +89,6 @@ typedef struct
 static GtkWidget *dialog_win;
 
 /* General */
-static GtkWidget *center_new_windows_checkbutton;
 static GtkWidget *allow_tiling_checkbutton;
 static GtkWidget *show_tab_border_checkbutton;
 static GtkWidget *titlebar_layout_optionmenu;
@@ -302,6 +301,7 @@ response_cb (GtkWidget *dialog_win,
 int
 main (int argc, char **argv)
 {
+    GtkWidget *center_new_windows_checkbutton;
     GError     *error = NULL;
     GtkBuilder *builder;
     GdkScreen  *screen;
```

Reply:
I think it is better to have them all in one place

Reply:
I mean there are some GtkWidget* which can't be seen as attributes of a class defined by a template, since they are not used beyond initialization, but rather as temporary variables that don't need to be stored in a permanent memory area as a global variables.

Reply:
It keeps consistency with existing code, so this change can be discussed later in a new merge proposal.